backend/imap/conn.go

```diff
@@ -31,39 +43,34 @@ type conns struct {
 }
 
 func (b *conns) connect(username, password string) (email string, err error) {
-	var c *imap.Client
+	var c *imapclient.Client
 	if b.config.Tls {
-		c, err = imap.DialTLS(b.config.Host(), nil)
+		c, err = imapclient.DialTLS(b.config.Host(), nil)
 	} else {
-		c, err = imap.Dial(b.config.Host())
+		c, err = imapclient.Dial(b.config.Host())
 	}
 	if err != nil {
 		return
 	}
 
-	//c.SetLogMask(imap.LogAll)
-
 	if !b.config.Tls {
```

Comment:
What do you mean? This checks if TLS is *not* enabled, and uses STARTTLS if so. Using non-encrypted connections is not secure!